app/views/portals/index.vue

```diff
@@ -1,19 +1,155 @@
+import { onMounted, ref, Ref } from 'vue';
+// @ts-expect-error importing js file
+import { formatDate } from '~/services/datetime';
+// @ts-expect-error importing js file
+import { lstring } from '~/components/kb/filters';
+// @ts-expect-error importing js file
+import { mapObjectId, isObjectId } from '~/api/api-base.js';
+// @ts-expect-error importing js file
+import ArticlesApi from '../../components/kb/article-api';
+// @ts-expect-error importing js file
+import PortalsApi from '~/api/portals'
+// @ts-expect-error importing js file
+import { useAuth } from '@scbd/angular-vue/src/index.js';
+// @ts-expect-error importing js file
+import { useRealm  } from '~/services/composables/realm.js';
+// @ts-expect-error importing js file
+import { useI18n } from 'vue-i18n'
 
-export default {
-  name:'onlineConferences',
-  components:{
-  },
-  computed: {
-  }
 
+const { realm } = useRealm()
+
+const { locale } = useI18n()
+
+type ContentSchema = {
+  article: { articleId: string }
+}
+type ArticleSchema = {
+  [key: string]: string | boolean | number | ContentSchema
 }
+type Image = { position: string, size: string, url: string }
+
+type Article = {
+  _id: string
+  coverImage: Image
+  summary: { en: string }
+  title: { en: string }
+  content: { en: string }
+  meta: { createdOn: string }
+  adminTags?: string[]
+  url?: string
+}
+type URLS_LIST = {
+  [key: string]: 'bch-iac' | 'capacity-building'
+}
+
+const auth = useAuth();
+const articlesApi = new ArticlesApi({tokenReader:()=>auth.token()});
+const portalsApi = new PortalsApi();
+const articles :Ref<Article[]> = ref([])
+
+// TODO: Set article adminTag values for the below bch portal articles in the database to avoid
+// hard-coding these URLs.
+const FORUM_URLS :URLS_LIST = {
+  '64d54b6b8e4a59a51e799916': 'bch-iac', // Set article.adminTags = [portal:bch-iac] in the DB
+  '6866bc79b7c31f956fd986b9': 'capacity-building' // Set article.adminTags = [portal:capacity-building] in the DB
+}
+const PORTAL_TAGS_STRING = 'portal:'
+const PORTALS_URL = 'portals'
+
+onMounted(async () => {
+  // http://localhost:2030/api/v2023/portals?q={"realms": "realm"}
+  const portals = await portalsApi.queryPortals({ q: { realms: realm } })
+
+  const articleOidQueries = portals
+    .filter((article: ArticleSchema) => isObjectId(article['_id']))
+    .map((article: ArticleSchema) => ({ $oid: mapObjectId((article['content'] as ContentSchema).article.articleId) }))
+
+  const query = [{ $match: {_id: { $in: articleOidQueries } } }]
+
+  const articleData = await articlesApi.queryArticles({ ag: JSON.stringify(query) })
+
+  articles.value = articleData.map((article: Article) => {
+    if (!article.adminTags) {
+      article.url = `${PORTALS_URL}/${FORUM_URLS[article._id]}`
+      return article
+    }
+
+    const portalTag = (article.adminTags || [])
+      .find(tag => tag.includes(PORTAL_TAGS_STRING)) || ''
+    article.url = `${PORTALS_URL}/${portalTag.replace(PORTAL_TAGS_STRING, '')}`
```

Comment:
use encodeURIComponent

```suggestion
    article.url = `${PORTALS_URL}/${encodeURIComponent(portalTag.replace(PORTAL_TAGS_STRING, ''))}`
```